internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,64 @@
 package daysteps
 
 import (
-	...
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"final5/internal/personaldata"
+	"final5/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) error {
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return fmt.Errorf("invalid data format")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return fmt.Errorf("invalid steps value: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("invalid steps value: %w", err)
```